app/v1/controllers/requestParsers/validators/SubmitValidator.scala

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.controllers.requestParsers.validators
+
+import v1.controllers.requestParsers.validators.validations.{MonetaryValueFormatValidation, MonetaryValueRangeValidation, PeriodKeyValidation, VrnValidation}
+import v1.models.errors.MtdError
+import v1.models.request.submit.{SubmitRawData, SubmitRequestBody}
+
+class SubmitValidator extends Validator[SubmitRawData] {
```

Comment:
Could this be named consistently with the request parser? Either have them both be Submit___ or SubmitReturn__